### DISCUSSION TITLE
Concerns about publicy hosted Vaultwarden's

### DISCUSSION THREAD

**Discussion by secwarz** [dani-garcia/vaultwarden#3810] [Ideas]
As a newcomer to this project, I have been looking around the internet about it and came across this website: vaultwarden.xx . Initially, I naively assumed that the site might be affiliated with the project creators. However, in adherence to good security practices, I habitually verify affiliations from both parties involved. Upon conducting this due diligence, I found no evidence to suggest that the aforementioned website is officially connected to this project, apart from the name similarity.

Given that this is open-source software, I recognize that any individual can deploy an instance of it online and make it accessible to the public. Nevertheless, it raises concerns when dealing with a password manager application. Specifically, the issue is that any entity can operate the software under its original name, but as a closed service on a website, potentially compromising security.

Consequently, I recommend that there be explicit indications about the official channels endorsed by the project. Additionally, there should be a warning regarding unofficial services that exploit the software's name and functionality.

**Comment by kpfleming** [comment]
There are no 'official channels', in the sense of publicly-available service offerings using Vaultwarden. This is not really different from any other open source software project, if there are hosted versions made available it's up to the user to decide whether they want to make use of them or not.

At best, the README could include a note stating that the project does not operate, support, or endorse any hosted service using the software.

**Comment by BlackDex** [comment]
To be fair, if we were connected to anything officially we probably would have mentioned it somewhere in the Readme, Discussions, Wiki, Matrix Channel, etc.. etc.. etc.. Which we do not.

Ill make it clear here, we do **not** host any version of Vaultwarden publicly, any public hosted version of Vaultwarden has nothing to do with this project other than that it uses this projects code.
